gateway/gateway.pb.go

```diff
@@ -2155,6 +1798,282 @@ func encodeVarintGateway(dAtA []byte, offset int, v uint64) int {
 	dAtA[offset] = uint8(v)
 	return offset + 1
 }
+func NewPopulatedLocationMetadata(r randyGateway, easy bool) *LocationMetadata {
```

Comment:
We don't need these, right?

Reply:
We agreed to use the same protoc as `v3`, hence this.